README.rst

```diff
@@ -82,6 +82,17 @@ Required
 .. |yosys| replace:: ``yosys``
 .. _yosys: https://github.com/YosysHQ/yosys
 
+By default, `verilog-diagram` uses the `yowasp-yosys` package provided in PyPI. It can be installed by running `pip install -r requirements.txt`.
+However, you could also use the `yosys` that is installed on your system, by adding the following line in `setup(app)` inside conf.py.
+
+.. code-block:: py
+
+   def setup(app):
+      ...
+      VerilogDiagram.use_yowasp = False
```

Comment:
In my opinion, the global settings for the extension should be based on variables added to sphinxconf.py file.
See: https://www.sphinx-doc.org/en/master/extdev/appapi.html#sphinx.application.Sphinx.add_config_value